### ISSUE TITLE
Thaumcraft focal manipulator not working

### ISSUE THREAD

**Issue by nostlie** [xJon/The-1.12.2-Pack#837]
I can create the focus with the Thaumcraft focal manipulator but the result disappears when I try to add it to my inventory.

**Comment by xJon** [comment]
Can you share a log with https://mclo.gs? When it disappears it would probably print an error to the log.

**Comment by nostlie** [comment]
Timestamp 2022/11/07 13:52:52 is about when I started crafting a Foci with the Focal Manipulator. After it finished crafting, I moved it to my inventory and it disappeared.

https://mclo.gs/h8dVKU1

Let me know if you need more info, and thanks for the fantastic modpack!

**Comment by xJon** [comment]
I can definitely see an error there! And this started to happen only in the latest version (`1.5.2`) of the modpack?

**Comment by nostlie** [comment]
I created a new `1.5.2` world in creative mode and all things thaumcraft seem to work fine, including the Focal Manipulator, unable to reproduce the issue. 

The multiplayer server that was upgraded from `1.5.1` to `1.5.2` is where I'm experiencing the issue. I will work with the server admin to see about downgrading to test that way.

**Comment by xJon** [comment]
Someone just told me they had this issue occur once (in a development environment). Not sure exactly what's the cause, but it could be a rare edge case?

**Comment by nostlie** [comment]
We did a bunch of troubleshooting, and were ultimately unable to reproduce this issue.

Took our `1.5.2` world that had the issue and downgraded to `1.5.1`. Tested things and everything was good. Then upgraded again to `1.5.2` and things still worked. So we're all good.

The only thing that I can think of that might be a cause, is that I had an electro-magic omnitool in my inventory when the upgrade occurred, and the Electro-Magic mod was removed from `1.5.1` to `1.5.2`. However, we tried to recreate this issue by giving ourselves an elctro-magic tool in a fresh `1.5.1` world then upgrading to see what happened. We were still unable to recreate the issue. 

I will mark this as solved and put it down as random modpack upgrade issues. Thanks for your attention and for the modpack!
